utils/CMakeLists.txt

```diff
@@ -1,8 +1,8 @@
 add_subdirectory(vim)
 
-if(SWIFT_INSTALL_TOOLS)
+# if(SWIFT_INSTALL_TOOLS)
```

Comment:
Agreed. If there is indeed something we can do better with regards to installing `gyb`, we should document that in a ticket on https://bugs.swift.org (along with *much* more detail than present here).